cmd/telegraf/telegraf.go

```diff
@@ -45,6 +47,8 @@ var fOutputFiltersLegacy = flag.String("outputfilter", "",
 	"filter the outputs to enable, separator is :")
 var fConfigDirectoryLegacy = flag.String("configdirectory", "",
 	"directory containing additional *.conf files")
+var fService = flag.String("service", "",
+	"operate on the service")
```

Comment:
it'll still show up in the `--help` output though. I would just leave it and put in a `(windows only)` blurb in the doc.